File: Iniciante/3214.cpp
#include <iostream>
using namespace std;

int main() {
    int e, f, c, refri = 0, g;
    cin >> e >> f >> c;
    g = e + f;
    while(g >= c) {
        refri += g/c;
        g = g/c + g%c;
    }
    cout << refri << endl;
    return 0;
}

File: Iniciante/C#/3214.cs
using System;
class URI {
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        int e = int.Parse(input[0]);
        int f = int.Parse(input[1]);
        int c = int.Parse(input[2]);
        int refri = 0;
        int g = e + f;
        while(g >= c) {
            refri += g/c;
            g = g/c + g%c;
        }
        Console.WriteLine(refri);
    }   
}